app/Helpers/StorageHelper.php

```diff
@@ -57,4 +57,14 @@ public static function hasReachedAccountStorageLimit(Account $account): bool
 
         return $currentAccountSize > (config('monica.max_storage_size') * 1000000);
     }
+
+    /**
+     * Get visibility to put files in storage.
+     *
+     * @return string
+     */
+    public static function visibility(): string
+    {
+        return config('filesystems.secure_files') ? 'private' : 'public';
```

Comment:
Why don't you simply return the config variable, and define a sensible default for the variable on the config file instead of making this comparison here?

Reply:
you mean define a config to be `private` or `public` (other values would probably not work ...)?
That's possible